metadata/src/test/java/org/apache/kafka/controller/PartitionChangeBuilderTest.java

```diff
@@ -77,39 +77,75 @@ public void testChangeRecordIsNoOp() {
         );
     }
 
-    private final static PartitionRegistration FOO = new PartitionRegistration(
-        new int[] {2, 1, 3}, new int[] {2, 1, 3}, Replicas.NONE, Replicas.NONE,
-        1, LeaderRecoveryState.RECOVERED, 100, 200);
+    private static final PartitionRegistration FOO;
+
+    static {
+        try {
+            FOO = new PartitionRegistration.Builder().
+                setReplicas(new int[] {2, 1, 3}).
+                setIsr(new int[] {2, 1, 3}).
+                setRemovingReplicas(Replicas.NONE).
+                setAddingReplicas(Replicas.NONE).
```

Comment:
We can remove this if we make the default value for these two fields `Replicas.NONE`.

Reply:
Done here and all the other places.